Makefile

```diff
@@ -39,6 +45,14 @@ serve:
 	-pkill -9 -f "node js/dev-server.js"
 	node js/dev-server.js
 
+serve.dev:
+	-pkill -9 -f "node js/dev-server.js"
+	APP_ENV=development node js/dev-server.js
+
+dev:
+	-pkill -9 -f "node js/dev-server.js"
+	npm-run-all --parallel build:dev serve:dev
```

Comment:
@ameshkin I wonder if adding "npx" in front here might let it work on certain environments, ie "npx npm-run-all..."